packages/reakit/src/Toolbar/__examples__/ToolbarWithPopover/__tests__/index-test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { render, press } from "reakit-test-utils";
+import ToolbarWithPopover from "..";
+
+const popoverText =
+  "This trip lasts three days and takes place in the otherworldly environment of Ares Station.";
```

Comment:
Instead of querying by the text we can maybe use the aria label and `getByLabelText` instead.